config/jobs/kubernetes/sig-api-machinery/sig-api-machinery-config.yaml

```diff
@@ -24,3 +24,28 @@ periodics:
   annotations:
     testgrid-dashboards: google-gce
     testgrid-tab-name: gci-gce-proto
+
+- interval: 24h
+  name: ci-kubernetes-e2e-gci-gce-network-proxy
```

Comment:
This job does not specify `decorate: true` (nor does the one above it) so it won't use the pod utilities and is instead using the deprecated bootstrap.py script. From the job logs for the above job:
```
W1205 22:20:31.483] **************************************************************************
bootstrap.py is deprecated!
test-infra oncall does not support any job still using bootstrap.py.
Please migrate your job to podutils!
https://github.com/kubernetes/test-infra/blob/master/prow/pod-utilities.md
**************************************************************************
```
I'll approve config changes, but if you need additional support with this job you'll need to migrate away from the deprecated bootstrap.py script first.